src/containers/edit-profile/professional-info-tab/professional-category/ProfessionalCategory.tsx

```diff
@@ -74,77 +70,54 @@ const ProfessionalCategory: FC<ProfessionalCategoryProps> = ({
     )
   }
 
-  const ToolbarButtons = item.isActivated ? (
-    <>
-      <AppButton
-        onClick={() => openProfessionalCategoryModal(item)}
-        variant={ButtonVariantEnum.ContainedLight}
-      >
-        {t('editProfilePage.profile.professionalTab.editCategoryBtn')}
-      </AppButton>
-      <Tooltip
-        arrow
-        placement={PositionEnum.Right}
-        title={
-          <Typography sx={styles.toolbar.deactivateButtonTooltip}>
-            {t(
-              `editProfilePage.profile.professionalTab.deactivateCategoryBtn${
-                !item.isActivationBlocked ? 'Enabled' : 'Disabled'
-              }Tooltip`
-            )}
-          </Typography>
-        }
-      >
-        <Box>
-          <AppButton
-            disabled={item.isActivationBlocked}
-            onClick={handleDeactivateButtonClick}
-            variant={ButtonVariantEnum.Tonal}
-          >
-            {t('editProfilePage.profile.professionalTab.deactivateCategoryBtn')}
-          </AppButton>
-        </Box>
-      </Tooltip>
-    </>
-  ) : (
-    <AppButton
-      onClick={handleActivateButtonClick}
-      variant={ButtonVariantEnum.Tonal}
-    >
-      {t('editProfilePage.profile.professionalTab.activateCategoryBtn')}
-    </AppButton>
-  )
-
   const Subjects = item.subjects.map((subject) => (
     <Box key={subject._id} sx={styles.subjects.item}>
       <CardItem label={t('editProfilePage.profile.professionalTab.subject')}>
         {subject.name}
       </CardItem>
-      <CardItem
-        label={t('editProfilePage.profile.professionalTab.proficiencyLevels')}
-      >
-        {subject.proficiencyLevels.join(', ')}
-      </CardItem>
     </Box>
   ))
 
   return (
-    <Box sx={styles.root(item.isActivated)}>
+    <Box sx={styles.root}>
       <Box sx={styles.toolbar.root}>
-        <Box sx={styles.toolbar.buttonGroup}>{ToolbarButtons}</Box>
-        <IconButton
-          data-testid='delete-professional-category-button'
-          onClick={handleDeleteButtonClick}
-          sx={styles.toolbar.deleteButton}
+        <Box sx={styles.toolbar.buttonGroup}>
+          <AppButton
+            onClick={() => openProfessionalCategoryModal(item)}
+            variant={ButtonVariantEnum.ContainedLight}
+          >
+            {t('editProfilePage.profile.professionalTab.editCategoryBtn')}
+          </AppButton>
+        </Box>
+        <Tooltip
+          arrow
+          placement={PositionEnum.Right}
+          sx={styles.toolbar.deactivateButtonTooltip}
+          title={
+            item.isDeletionBlocked
+              ? t(
+                  'editProfilePage.profile.professionalTab.deleteCategoryBtnDisabledTooltip'
+                )
+              : ''
```

Comment:
can we refactor to this syntax?

`item.isDeletionBlocked &&  t( 'editProfilePage.profile.professionalTab.deleteCategoryBtnDisabledTooltip')`

Reply:
Yes we can) thank you